CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ FEATURES:
 IMPROVEMENTS:
 * Make `server.bootstrapExpect` optional. If not set, will now default to `server.replicas`.
   If you're currently setting `server.replicas`, there is no effect. [[GH-721](https://github.com/hashicorp/consul-helm/pull/721)]
+* Enable client agents outside of the K8s cluster to join the consul datacenter
```

Comment:
I think this is a feature? Pretty useful.

Maybe mention the name of the setting? Like "Added new Helm values `server.exposeGossipAndRPCPorts` and `server....`

Reply:
Cool, moved it to feature and mentioned the helm values!